source/systems/worker.md

```diff
@@ -76,6 +76,8 @@ A chart of all the jobs and their traits, from hog(#1112) on [Discord](https://d
 | [Undertaker](../workers/undertaker) | Strength | Mana |
 | [University Researcher](../workers/researcher) | Knowledge | Mana |
 
+<p style="text-align:center; font-size:20pt;"><img src="../../assets/images/misc/workerTraitEffects.jpg" alt="Sleep"><b> From Estyria#2729 on [Discord](https://discord.minecolonies.com):</b></p>
```

Comment:
I don't see the image uploaded? How would this work?

Reply:
Oops. Did the image commit not get included in the PR?

Reply:
Nope, apparently not

Reply:
Now it should.